spring-boot-project/spring-boot-autoconfigure/src/main/resources/META-INF/additional-spring-configuration-metadata.json

```diff
@@ -1677,6 +1677,18 @@
         "replacement": "spring.reactor.debug-agent.enabled"
       }
     },
+    {
```

Comment:
This should move in code so that the metadata is generated automatically.

`Jedis#Pool` should have a `boolean` enabled field that's true by default.
`Lettuce#Pool` should have a `Boolean` enabled field that has no default value.